examples/plot_mne_dataset_example.py

```diff
@@ -11,7 +11,7 @@
 
 import mne
 
-from braindecode.datasets.mne import (
+from braindecode.datautil.mne import (
```

Comment:
```suggestion
from braindecode.datautil import (
```

? 

to avoid too deep imports...